src/celeritas/em/data/WentzelData.hh

```diff
@@ -0,0 +1,113 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/data/WentzelData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/data/Collection.hh"
+#include "celeritas/Quantities.hh"
+#include "celeritas/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Particle and action ids used by WentzelModel
+ */
+struct WentzelIds
+{
+    ActionId action;
+    ParticleId electron;
+    ParticleId positron;
+
+    explicit CELER_FUNCTION operator bool() const
+    {
+        return action && electron && positron;
+    }
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Per-element data used by the WentzelModel
+ */
+struct WentzelElementData
+{
+    // Matrix of Mott coefficients
+    real_type mott_coeff[5][6];
```

Comment:
Even better would be the `Array` type (see `LivermorePEData.hh`):
```suggestion
    using using Real6 = Array<real_type, 6>;

     //! Matrix of Mott coefficients[theta][beta]
    Array<Real6, 5> mott_coeff;
```

Reply:
Can I use any of the STL algorithms on arrays if they're not in `corecel/math/ArrayUtils.hh` or are for loops preferred?

Reply:
Please use what @sethrj suggested.
```
using Real6 = Array<real_type, 6>;
Array<Real6, 5> mott_coeff;
```

Reply:
Yep, the standard library algorithms will work with `Array` which has begin/end/size etc.

Reply:
Is it possible to get the size of an `Array` type at compile time? It would be useful to have something like
```
using BetaArray = Array<real_type, 6>;
using ThetaArray = Array<real_type, 5>;
Array<BetaArray, ThetaArray::size()> mott_coeff;
```
and then calculating the Mott ratio simplifies to doing some dot products between `BetaArray`s and `ThetaArray`s.

Reply:
Yes indeed, although it looks weird. Since `size` and the default constructor are both `constexpr`, you can use
```c++
Array<BetaArray, ThetaArray{}.size()> mott_coeff;
```
although this is less readable than Soon's suggestion, so I say we combine them:
```c++
inline static constexpr size_type num_beta_bins = 6;
inline static constexpr size_type num_theta_bins = 5;
using BetaArray = Array<real_type, num_beta_bins>;
using ThetaArray = Array<real_type, num_theta_bins>;

Array<BetaArray, num_theta_bins> mott_coeff;
```